src/components/Projects/ProjectsTabs.tsx

```diff
@@ -0,0 +1,68 @@
+import { t } from '@lingui/macro'
+import { Space } from 'antd'
+import { NetworkContext } from 'contexts/networkContext'
+import { ProjectCategory } from 'models/project-visibility'
+import React, { useContext } from 'react'
+
+export default function ProjectsTabs({
+  selectedTab,
+  setSelectedTab,
+}: {
+  selectedTab: ProjectCategory
+  setSelectedTab: Function
+}) {
+  const { signingProvider } = useContext(NetworkContext)
+
+  const tab = (tab: ProjectCategory) => {
+    let text: string = ''
+    // Need this switch for translations
+    switch (tab) {
+      case 'all':
+        text = t`All`
+        break
+      case 'myprojects':
+        text = t`My projects`
+        break
+      case 'trending':
+        text = t`Trending`
+        break
+    }
+    return (
+      <div
+        style={{
+          textTransform: 'uppercase',
+          cursor: 'pointer',
+          borderBottom: '2px solid transparent',
+          paddingBottom: 6,
+          ...(tab === selectedTab
+            ? {
+                color: 'var(--text-primary)',
+                fontWeight: 500,
+                borderColor: 'var(--text-primary)',
+              }
+            : {
+                color: 'var(--text-secondary)',
+                borderColor: 'transparent',
+              }),
+        }}
+        onClick={() => {
+          // Remove ?trending if we came from homepage trending section
+          window.location.href = `/#/projects?tab=${tab}`
```

Comment:
**suggestion (non-blocking)**: we can improve this by making the URL path the single source of truth, and leverage `react-router` API to handle the tab selection: https://v5.reactrouter.com/web/api/Hooks/useroutematch

Don't want to block on this if the current implementation is working, but just a thought.

Reply:
Good idea, we'll clean that up later